torch/onnx/_constants.py

```diff
@@ -4,7 +4,7 @@
 
 ONNX_BASE_OPSET = 9
 ONNX_MIN_OPSET = 7
-ONNX_MAX_OPSET = 19
+ONNX_MAX_OPSET = 20
 ONNX_TORCHSCRIPT_EXPORTER_MAX_OPSET = 17
```

Comment:
```suggestion
ONNX_TORCHSCRIPT_EXPORTER_MAX_OPSET = 20
```

This bumps version of both torchscript exporter's and unittests